lib/open_api/renderer/operation.ex

```diff
@@ -381,9 +383,11 @@ defmodule OpenAPI.Renderer.Operation do
     end
   end
 
-  defp render_return_type([], _type_overrides), do: quote(do: :ok)
+  defp render_return_type(_state, []), do: quote(do: :ok)
```

Comment:
Sorry for going outside of this PRs limits, but shouldn't this clause check for the empty list on the second argument?